### PR TITLE
feature/tslint config

### PR DESCRIPTION
**Pull request by smollweide**

### CODE REVIEW

File: codequality/tslint-config/src/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const rimraf = require('rimraf');
+const rules = require('./rules');
```
[the remark this answers is not in the document]
Comment (smollweide) [reply codequality/tslint-config/src/index.js]: I renamed it to tsRules

**Comment by smollweide** [comment]
What do you think about [tslint-clean-code](https://github.com/Glavin001/tslint-clean-code), should we add it as default?

**Comment by smollweide** [comment]
@ernscht: thank you.
I fixed the package-lock and the tests.
Also I updated/changed some rules. Also some more tests were added.

**Comment by smollweide** [comment]
@ernscht, @jantimon:
I changed some rules:
- [jsx-boolean-value](https://github.com/namics/frontend-defaults/blob/feature/tslint-config/codequality/tslint-config/src/reactRules.js#L5)
- [jsx-no-lambda](https://github.com/namics/frontend-defaults/blob/feature/tslint-config/codequality/tslint-config/src/reactRules.js#L15)
- [jsx-no-multiline-js](https://github.com/namics/frontend-defaults/blob/feature/tslint-config/codequality/tslint-config/src/reactRules.js#L17)

- [variable-name](https://github.com/namics/frontend-defaults/blob/feature/tslint-config/codequality/tslint-config/src/tsRules.js#L230)

**Comment by jantimon** [comment]
Looks really good 👍

**Pull request merged**